src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -51,4 +51,11 @@ md-toolbar {
       }
     }
   }
+
+  // Fixes pickers placement issue on Internet Explorer.
+  .md-select-value {
+    *:first-child{
+      -ms-transform: translateY(0.75px);
```

Comment:
You will probably ask, why exactly `0.75px` and honestly I'm not sure. I know it's only value, which makes it look good. I don't understand IE, and why translate works different here.